app/assets/javascripts/discourse/components/category-group.js.es6

```diff
@@ -1,37 +0,0 @@
-import { categoryBadgeHTML } from 'discourse/helpers/category-link';
```

Comment:
I saw other autocomplete components are using `-selector` so I think it's better to keep them consistent. Others are `user-selector` and `group-selector`